### PR TITLE
Added default gate type to created fault tree

### LINKED ISSUE #346: Creating a new Fault Tree using a created event does not work
When creating a new Fault Tree with a new Event (see screenshot), the editor does not work:

Image

Image

### PR DESCRIPTION
**Pull request by LaChope**
Resolves #346.

### CODE REVIEW

File: src/components/dialog/faultEvent/FaultEventCreation.schema.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const schema = Yup.object().shape({
     .max(1, "Sequence probability cannot be greater than 1"),
 });
 
-export const rootEventSchema = Yup.object().shape({
+export const eventSchema = Yup.object().shape({
```
Comment (blcham) [inline src/components/dialog/faultEvent/FaultEventCreation.schema.tsx:39]: Do not understand the code, but isn't it useful to know it is related to root event.

**Review by blcham — APPROVED** [review]
Can be merged, just check my comment

File: src/components/dialog/faultEvent/FaultEventCreation.tsx [hunk 2]
```diff
@@ -168,20 +168,18 @@ const FaultEventCreation = ({ useFormMethods, isRootEvent }: Props) => {
...
-                max={1}
-                step={0.01}
-                error={!!errors.probability}
-                helperText={errors.probability?.message}
-                className={classes.probability}
-                defaultValue=""
-                {...register("probability")}
-              />
-            )}
+            <TextField
```
Comment (blcham) [inline src/components/dialog/faultEvent/FaultEventCreation.tsx:171]: @LaChope I do not understand this. Why would you want to set up probability for intermediate event ?

File: src/components/dialog/faultEvent/FaultEventCreation.schema.tsx (hunk 1 above)
Comment (LaChope) [reply src/components/dialog/faultEvent/FaultEventCreation.schema.tsx:39]: FaultEventSchema

**Comment by LaChope** [comment]
@blcham I am not merging yet because I am using this branch to also resolve https://github.com/kbss-cvut/fta-fmea-ui/issues/352.

**Comment by blcham** [comment]
> @blcham I am not merging yet because I am using this branch to also resolve #352.

ok

**Comment by LaChope** [comment]
@blcham this is ready for review now

File: src/components/dialog/faultEvent/FaultEventCreation.tsx (hunk 2 above)
Comment (blcham) [reply src/components/dialog/faultEvent/FaultEventCreation.tsx:177]: @LaChope you resolved this but did not provide info how

**Review by blcham — APPROVED** [review]
LGTM, but I was not able to check the PR. It is too complicated to me (seems like a lot of reformating of code but was not able to identify which code does just reformating and which is actually doing some work)

File: src/components/dialog/faultEvent/FaultEventCreation.tsx (hunk 2 above)
Comment (LaChope) [reply src/components/dialog/faultEvent/FaultEventCreation.tsx:177]: We resolved it during Stand-up but yes I did not write it here: We agreed that Intermediate and external events should not have probability, all other event types should have one

Comment (LaChope) [reply src/components/dialog/faultEvent/FaultEventCreation.tsx:177]: Resolved by https://github.com/kbss-cvut/fta-fmea-ui/pull/396.

**Pull request merged**